features/utils.js

```diff
@@ -21,10 +21,14 @@ const authHeader = new Buffer.from(
 ).toString('base64')
 
 const patientId = '12305759751'
-const patientId2 = '1234567'
+const patientId1 = '1234567'
```

Comment:
The `new Buffer.from(` line above has a warning from VS Code. The Buffer object can be used directly - you don't need `new`

Reply:
addressed in https://github.com/jembi/who-covid19-surveillance-package/pull/19/commits/1d40f20f1217c21e38dde9b0fdc4ae6cf9b61d12